workshop_app/test_models.py

```diff
@@ -0,0 +1,190 @@
+from django.test import TestCase
+from .models import (
+					Profile, User, Workshop, WorkshopType, 
+                    RequestedWorkshop, BookedWorkshop, ProposeWorkshopDate
+                    )
+from datetime import datetime
+
+# Setup for Model Test
+def setUpModule():
+	'''
+		demo user as coordinator and test user as instructor
+	'''
+
+	demoUser1 = User.objects.create(username='demouser1', 
+						email='doke.akshen@gmail.com', password='pass@123')
+	demoUser2 = User.objects.create(username='demouser2', 
+						email='doke.akshen@gmail.com', password='pass@123')
+
+	testUser1 = User.objects.create(username='testuser1',
+						email='doke.akshen@gmail.com',password='pass@123')
+
+	testUser2 = User.objects.create(username='testuser2',
+				email='doke.akshen@gmail.com', password='pass@123')
+
+	ip = Profile.objects.create(user=testUser2, position='instructor',
+			department='cs', institute='ace', phone_number='9930011492',
+			is_email_verified=1)
+
+	cp = Profile.objects.create(user=demoUser2, position='coordinator',
+			department='IT', institute='iit', phone_number='9930011492',
+			is_email_verified=1)
+
+	wt1 = WorkshopType.objects.create(workshoptype_name='ISCP',
+			workshoptype_description='Introduction to Scientific Computing in\
+			Python <br> > Numpy <br> > Matplotlib <br> > iPython <br>',
+			workshoptype_duration='1day, 8hours a day')
+
+	rw = RequestedWorkshop.objects.create(
+						requested_workshop_instructor=testUser2,
+						requested_workshop_coordinator=demoUser2,
+						requested_workshop_title=wt1,
+						requested_workshop_date='2017-07-24'
+						)
+	
+	pw = ProposeWorkshopDate.objects.create(
+					proposed_workshop_coordinator=demoUser2,
+					proposed_workshop_instructor=testUser2,
+					proposed_workshop_title=wt1,
+					proposed_workshop_date='2017-07-06',
+					condition_one=1,
+					condition_two=1,
+					condition_three=1
+					)
+
+
+def tearDownModule():
+    User.objects.all().delete()
+    Profile.objects.all().delete()
+    ProposeWorkshopDate.objects.all().delete()
+    RequestedWorkshop.objects.all().delete()
+    WorkshopType.objects.all().delete()
+
+class ProfileModelTest(TestCase):
+	'''	
+		This class tests the Profile Model
+	'''
+	def setUp(self):
+		'''
+			ip is for instructor profile and cp is for coordinator profile
+		'''	
+		self.testuser1 = User.objects.get(username='testuser1')
+		self.demouser1 = User.objects.get(username='demouser1')
+		
+
+		self.ip1 = Profile.objects.create(user=self.testuser1, position='instructor',
```

Comment:
These could be <code>self.instructor1</code> and <code>self.coordinator1 </code>